### PR TITLE
[FIX] mail: display the call indicator on mobile

### PR DESCRIPTION
**Pull request by phenix-factory**
and in the notification zone.

part of task-2634175

### CODE REVIEW

File: addons/mail/static/src/components/thread_preview/thread_preview.xml [hunk 1]
```diff
@@ -39,6 +39,10 @@
                                 <t t-esc="thread.lastMessage.date.fromNow()"/>
                             </span>
                         </t>
+                        <t t-if="thread and thread.rtcSessions.length">
+                            <span class="o-autogrow"/>
```
Comment (seb-odoo) [inline addons/mail/static/src/components/thread_preview/thread_preview.xml:43]: I would not move the date on the middle, this is really strange. I would suggest to move the icon somewhere else, probably just before the existing `o-autogrow`?

File: addons/mail/static/src/components/thread_preview/thread_preview.xml [hunk 2]
```diff
@@ -39,6 +39,10 @@
                                 <t t-esc="thread.lastMessage.date.fromNow()"/>
                             </span>
                         </t>
+                        <t t-if="thread and thread.rtcSessions.length">
```
Comment (seb-odoo) [inline addons/mail/static/src/components/thread_preview/thread_preview.xml:42]: Use explicit value when comparing numbers in conditions (`> 0` in this case?)

Comment (seb-odoo) [reply addons/mail/static/src/components/thread_preview/thread_preview.xml:42]: Also condition on thread is already done at the top of the template.

**Review by seb-odoo — CHANGES_REQUESTED** [review]
It needs a bit of (left) margin, and better vertical alignment with the text.
image

**Comment by seb-odoo** [comment]
Also it needs right margin in overflow situations (channel name too long)

**Review by seb-odoo — CHANGES_REQUESTED** [review]
Looking ok visually, some remarks about the code

**Comment by phenix-factory** [comment]
@seb-odoo I have updated this PR

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Review by seb-odoo — APPROVED** [review]
rebased to get the fix for runbot

@robodoo r+

**Pull request closed without merging**